common/ustring.h

```diff
@@ -25,7 +25,13 @@ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
 #define UString CBString
 #endif // QT_CORE_LIB
 
-UString usprintf(const char* fmt, ...);
+#if defined(_MSC_VER)
```

Comment:
Cannot we reuse that macro from bstrlib? If we cannot, I believe it should be put to a separate header.